app/templates/briefs/clarification_question.html

```diff
@@ -0,0 +1,99 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/summary-table.html" as summary %}
+
+{% block page_title %}Ask a question about {{ brief.title }} – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace",
+      },
+      {
+        "link": url_for(".dashboard"),
+        "label": "Your account",
+      },
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="single-question-page">
+  {% with messages = get_flashed_messages(with_categories=true) %}
+    {% for category, message in messages %}
+      {% if message == 'message_sent' %}
+        {% set message = "Your question has been sent. The answer will be published on the opportunity page." %}
+      {% endif %}
+      {%
+        with
+        message = message,
+        type = "destructive" if category == 'error' else "success"
+      %}
+        {% include "toolkit/notification-banner.html" %}
+      {% endwith %}
+    {% endfor %}
+  {% endwith %}
+
+  {% if error_message %}
+    {%
+      with
+      errors = [
+        {
+            "input_name": clarification_question_name,
+            "question": "Ask a {} clarification question".format(brief.title)
+        }
+    ],
+      lede = "There was a problem with your submitted question"
+    %}
+      {% include "toolkit/forms/validation.html" %}
+    {% endwith %}
+  {% endif %}
+
+  {% with
+    heading = "Ask a question about {}".format(brief.title),
```

Comment:
We're using [smart curly Ralph single quotes](https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/pc_supplier_reply_responses/app/main/views/briefs.py#L40) in the title for the supplier's brief response.